docs/src/imagesegmentation.md

```diff
@@ -451,6 +451,8 @@ julia> segments = watershed(dist, markers)
 Segmented Image with:
   labels map: 312×252 Array{Int64,2}
   number of labels: 24
+
+julia> segments = imshow( map(i->segment_mean(segments,i), labels_map(segments)) .* .~bw)       #shows segmented image
```

Comment:
I don't know this code snippet, but the `segments = ...` prefix doesn't quite look right

Reply:
@Evizero Updated. The code snippet is from the docs only, I have just added a mask to show the segmented image.